juspay/hyperswitch
# feat(braintree): Sessions flow for braintree and klarna

## Type of Change



## Description

Implement connector integration for braintree to retrieve sessions token for paypal.


## Motivation and Context

This feature will enable to create session token for paypal through braintree.


## How did you test it?
- Create a merchant account ( connectors pecking order does not matter for sessions token ).
-  Create klarna account.
- Create braintree account.
- First create a payment with confirm set to false.
- Hit `/payments/session_tokens` endpoint with this body
`{
    "payment_id": ,
    "client_secret": 
}`

<img width="1248" alt="Screenshot 2022-12-12 at 5 06 07 PM" src="https://user-images.githubusercontent.com/48803246/207035602-98559540-2768-4d35-b53d-4814e167182b.png">


## Checklist


- [x] I formatted the code `cargo +nightly fmt`
